SensioLabsConnectBundle.php

```diff
@@ -27,7 +27,8 @@ class SensioLabsConnectBundle extends Bundle
 {
     public function build(ContainerBuilder $container)
     {
-        if ($container->has('security.authentication.provider.sensiolabs_connect')) {
+        if ($container->hasExtension('security')) {
```

Comment:
But we now miss a way to disable connect security without disabling all the security system :/